Lib/sqlite3/test/factory.py

```diff
@@ -183,6 +183,14 @@ def CheckSqliteRowHashCmp(self):
         self.assertNotEqual(row_1, row_3)
         self.assertNotEqual(hash(row_1), hash(row_3))
 
+    def CheckSqliteRowRepr(self):
+        self.con.row_factory = sqlite.Row
+        row = self.con.execute(
+            "select 'Smith' as name, 42 as salary").fetchone()
```

Comment:
Hi! why don't use capitalize to sql sentences: `SELECT 'Smith' As name 42 As salary` ?

Reply:
@eamanu all the queries are in lowercase in this file `Lib/sqlite3/test/factory.py`

Reply:
I would prefer CAPS myself, but yeah, used lowercase for consistency.